Resources/doc/reference/filter_field_definition.rst

```diff
@@ -90,6 +90,19 @@ This form type requires ``property`` option. See documentation of ``sonata_type_
         ;
     }
 
+doctrine_orm_date_range
+-----------------------
+The ``doctrine_orm_date_range`` filter renders two fields to filter for all records between two dates. If only one date is set it will filter for all records up
```

Comment:
120 char limit applies here too. Be clever about it and add a newline after the sentence instead of in the middle of it.